test/helperTests.js

```diff
@@ -0,0 +1,54 @@
+var nohm = require(__dirname+'/../lib/nohm').Nohm;
+var args = require(__dirname+'/testArgs.js');
+var redis = args.redis;
+var helperLib = require(__dirname+'/../lib/helpers.js');
+var h = require(__dirname+'/helper.js');
+
+var UserLinkMockup = nohm.model('UserLinkMockup', {
+  properties: {
+    name: {
+      type: 'string',
+      defaultValue: 'testName',
+      validations: [
+        'notEmpty'
+      ]
+    }
+  },
+  idGenerator: 'increment'
+});
+var RoleLinkMockup = nohm.model('RoleLinkMockup', {
+  properties: {
+    name: {
+      type: 'string',
+      defaultValue: 'user'
+    }
+  },
+  idGenerator: 'increment'
+});
+
+
+exports.helpers = {
+  
+  setUp: function (next) {
+    if ( ! nohm.client) {
+      nohm.setClient(redis);
+    }
+    next();
+  },
+  tearDown: function (next) {
+    h.cleanUp(redis, args.prefix, next);
+  },
+
+  checkEqual: function(t) {
+    let role = new RoleLinkMockup();
```

Comment:
Small issue with this: the v1 branch is currently set to support node all the way back to v0.10 (stone ages).

That means no let/const here (should be const if it was allowed).